beniget/ordered_set.py

```diff
@@ -0,0 +1,90 @@
+"""
+Stolen from https://github.com/bustawin/ordered-set-37
```

Comment:
I think that "Stolen" is not the appropriate wording :-) "Copied" is probably better ;-)